python/__init__.py

```diff
@@ -16,6 +16,33 @@ def get_version():
         return version_file.read().strip()
 
 
+def modcc():
```

Comment:
Apart from the error handling: What's the motivation?
Shelling out is usually frowned upon